source/agora/common/crypto/Key.d

```diff
@@ -105,12 +105,18 @@ public struct PublicKey
     /*private*/ BitBlob!(crypto_sign_ed25519_PUBLICKEYBYTES * 8) data;
     alias data this;
 
-    /// Constructor accepting only 1 argument
+    /// Construct an instance from the binary representation
     private this (typeof(PublicKey.tupleof) args) pure nothrow @safe @nogc
     {
         this.tupleof = args;
     }
 
+    /// Ditto
+    public this (const ubyte[] args) pure nothrow @safe @nogc
```

Comment:
Unittest?

Reply:
It's called in the next commit

Reply:
Ah I got a taste of my own medicine just now.